packages/models/src/actions/code.ts

```diff
@@ -14,61 +11,90 @@ export enum CodeActionType {
     UNGROUP = 'ungroup',
 }
 
-interface BaseCodeAction {
-    type: CodeActionType;
-    location: ActionElementLocation;
-    uuid: string;
-}
+const BaseCodeActionSchema = z.object({
+    type: z.literal(CodeActionType.MOVE),
+    location: MoveActionLocationSchema,
```

Comment:
Neither is this. Seems like this should be ActionElementLocationSchema

Reply:
Change this to ActionElementLocationSchema causes circular dependency. Annoying...